viewer/packages/360-images/src/icons/Image360PointsObject.ts

```diff
@@ -0,0 +1,45 @@
+/*!
+ * Copyright 2023 Cognite AS
+ */
+import { OverlayPointsObject } from '@reveal/3d-overlays';
+import { PointsMaterialParameters } from '@reveal/3d-overlays/src/OverlayPointsObject';
+import { LessEqualDepth, GreaterDepth, Object3D, Vector3 } from 'three';
+
+export class Image360PointsObject extends Object3D {
+  private readonly _pointsFront: OverlayPointsObject;
+  private readonly _pointsBack: OverlayPointsObject;
+
+  constructor(
+    maxPoints: number,
+    materialParameters: Pick<
+      PointsMaterialParameters,
+      'spriteTexture' | 'minPixelSize' | 'maxPixelSize' | 'radius' | 'colorTint'
+    >
```

Comment:
Could we construct the `MaterialParameters` like its own type and then let `PointsMaterialParameters` extend it?

Reply:
Not sure why I added `Pick` here but it works normally with original `PointsMaterialParameters` type.